compose.yaml

```diff
@@ -48,3 +48,10 @@ services:
     extends:
       file: ./payments-be/compose.yaml
       service: payments-db
+
+  # game
+  game-service:
+    extends:
+      file: ./game-service/compose.yaml
```

Comment:
Should be `./game-service-be/compose.yaml`

Reply:
done